lib/manager/fvm/readme.md

```diff
@@ -0,0 +1 @@
+Simply keeps the `.fvm/fvm_config.json` file updated.
```

Comment:
The documentation for this new feature feels a bit light. Is this really all we need our users to know?

Can you put your `fvm_config.json` file anywhere you want, or must it be in the `.fvm` folder for `fvm` to work? If people can put the file _anywhere_ they like, maybe we need to mention that we only search in the `.fmv/*` folder?

Reply:
I copied this from the `nvm` manager [README](https://github.com/renovatebot/renovate/blob/main/lib/manager/nvm/readme.md). And I'm pretty sure the `fvm_config.json` file has to be under the `.fvm/` folder. I couldn't find anything on their docs mentioning a different location for the config.

Reply:
Oh okay! The maintainers can resolve this conversation then. 😉